smart_control/simulator/building.py

```diff
@@ -665,14 +714,25 @@ def __init__(
         and walls
       convection_simulator: object to simulate air convection
       reset_temp_values: Temp values to use when resetting the building
+      inside_air_radiative_properties: RadiationProperties for interior air.
+      inside_wall_radiative_properties: RadiationProperties for interior walls.
+      building_exterior_radiative_properties: RadiationProperties for building's
+        exterior.
+      include_radiative_heat_transfer: bool to note whether to include radiative
+        heat transfer.
+      view_factor_method: str to note the method to use for view factors.
+        Either "ScriptF" or "CarrollMRT". See
+        https://bigladdersoftware.com/epx/docs/9-6/engineering-reference/
+        inside-heat-balance.html#lw-radiation-exchange-among-zone-surfaces
```

Comment:
we can use a markdown link here to keep the link intact, and it should not violate line too long.